jet-fuel/test/controllers/urls_controller_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+
+class UrlsControllerTest < ActionController::TestCase
+  test "should create url" do
+    assert_difference('Url.count') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.